sections/futures/PositionCard/ClosePositionModalIsolatedMargin.tsx

```diff
@@ -0,0 +1,74 @@
+import useSynthetixQueries from '@synthetixio/queries';
+import { useEffect } from 'react';
+import { useRecoilValue } from 'recoil';
+
+import TransactionNotifier from 'containers/TransactionNotifier';
+import { useFuturesContext } from 'contexts/FuturesContext';
+import { useRefetchContext } from 'contexts/RefetchContext';
+import useEstimateGasCost from 'hooks/useEstimateGasCost';
+import { KWENTA_TRACKING_CODE } from 'queries/futures/constants';
+import { currentMarketState, futuresAccountTypeState, positionState } from 'store/futures';
+import { gasSpeedState } from 'store/wallet';
+
+import ClosePositionModal from './ClosePositionModal';
+
+type Props = {
+	onDismiss: () => void;
+};
+
+export default function ClosePositionModalIsolatedMargin({ onDismiss }: Props) {
+	const { handleRefetch } = useRefetchContext();
+	const { useEthGasPriceQuery, useSynthetixTxn } = useSynthetixQueries();
+	const ethGasPriceQuery = useEthGasPriceQuery();
+	const gasSpeed = useRecoilValue(gasSpeedState);
+	const { monitorTransaction } = TransactionNotifier.useContainer();
+	const { resetTradeState } = useFuturesContext();
+	const { estimateSnxTxGasCost } = useEstimateGasCost();
+
+	const currencyKey = useRecoilValue(currentMarketState);
+	const position = useRecoilValue(positionState);
+	const positionDetails = position?.position;
+	const selectedAccountType = useRecoilValue(futuresAccountTypeState);
+
+	const gasPrice = ethGasPriceQuery.data?.[gasSpeed] ?? null;
+
+	const closeTxn = useSynthetixTxn(
+		`FuturesMarket${currencyKey?.[0] === 's' ? currencyKey.substring(1) : currencyKey}`,
```

Comment:
I think the `getDisplayAsset` function in `utils/futures.ts` can do this.

Reply:
nice, will update